src/qmflows/parsers/cp2KParser.py

```diff
@@ -136,26 +177,27 @@ def remove_trailing(xs):
     # in block cotaining a maximum of two columns of MOs
     chunks = chunked(rs, nOrbFuns + 3)
 
-    eigenVals = np.empty(nOrbitals)
+    energies = np.empty(nOrbitals)
     coefficients = np.empty((nOrbFuns, nOrbitals))
 
     convert_to_float = np.vectorize(float)
```

Comment:
I don't actually think the additional `float` vectorization is necessary here:

``` python

>>> import numpy as np

>>> a = np.zeros((5, 5), dtype=float)
>>> a[0] = ['0', '1', '2', '3', '4']
>>> a
array([[0., 1., 2., 3., 4.],
       [0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0.]])
```

Reply:
Thanks!